### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
Due to breaking changes in the testing library `ava` tests where moved and renamed. 1 unit tests didn't work (flaky).

### CODE REVIEW

**Review by sjaanus — APPROVED** [review]
LG

File: package.json [hunk 1]
```diff
@@ -29,49 +29,55 @@
...
   "homepage": "https://github.com/Unleash/unleash-client-node",
   "dependencies": {
-    "ip": "^1.1.8",
-    "make-fetch-happen": "^10.2.1",
-    "murmurhash3js": "^3.0.1",
-    "semver": "^7.3.8"
+    "ip": "1.1.8",
+    "make-fetch-happen": "11.1.1",
+    "murmurhash3js": "3.0.1",
+    "semver": "7.5.1"
```
Comment (SimenB) [inline package.json:35]: why are you locking them down?

Comment (ivarconr) [reply package.json:35]: Good call @SimenB, I do not believe this was an intended change.

Comment (Tymek) [reply package.json:35]: @SimenB I'll get back to you when we decide what to do about pinning dependencies

Comment (Tymek) [reply package.json:35]: @SimenB updated in `v4.0.1`

Comment (SimenB) [reply package.json:35]: thanks!

**Pull request merged**